schemas/public/onpremises-kfd-v1alpha2.json

```
@@ -518,33 +539,34 @@
           "items": {
             "type": "string"
           },
-          "description": "The tls cipher suites to use"
+          "description": "The TLS cipher suites to use for etcd, kubelet, and kubeadm static pods. Example:\n```yaml\ntlsCipherSuites:\n  - \"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256\"\n  - \"TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256\"\n  - \"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384\"\n  - \"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384\"\n  - \"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256\"\n  - \"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256\"\n  - \"TLS_AES_128_GCM_SHA256\"\n  - \"TLS_AES_256_GCM_SHA384\"\n  - \"TLS_CHACHA20_POLY1305_SHA256\"\n```"
         },
         "configuration": {
           "type": "string",
-          "description": "The configuration to use"
+          "description": "etcd's encryption at rest configuration. Must be a string with the EncryptionConfiguration object in YAML. Example:\n\n```yaml\n\napiVersion: apiserver.config.k8s.io/v1\nkind: EncryptionConfiguration\nresources:\n  - resources:\n    - secrets\n    providers:\n    - aescbc:\n        keys:\n        - name: mykey\n          secret: base64_encoded_secret\n```\n"
         }
       }
     },
     "Spec.Kubernetes.Advanced.AirGap": {
       "type": "object",
+      "description": "Advanced configuration for air-gapped installations. Allows setting custom URLs where to download the binaries dependencies from and custom .deb and .rpm package repositories.",
       "additionalProperties": false,
       "properties": {
         "containerdDownloadUrl": {
           "type": "string",
-          "description": "The containerd download url"
+          "description": "URL where to download the containerd binary from."
```

Is this the binary or the tar.gz ?

---

excellent question, can you please confirm @alessiodionisi?

---

The tar.gz from GitHub.

---

```suggestion
          "description": "URL where to download the `.tar.gz` with containerd from. The `tar.gz` should be as the one downloaded from containerd GitHub releases page."
```

is this OK for you @alessiodionisi @nutellinoit ?

---

LGTM